modules/swagger-codegen/src/main/resources/lua/api.mustache

```diff
@@ -86,30 +75,30 @@ function {{classname}}:{{operationId}}({{#allParams}}{{#required}}{{paramName}}{
     {{/-last}}
     {{/formParams}}
     {{#bodyParams}}
-	req:set_body(dkjson.encode({{paramName}}))
+    req:set_body(dkjson.encode({{paramName}}))
 
     {{/bodyParams}}
     {{#authMethods}}
     {{#isApiKey}}
     {{#isKeyInHeader}}
-    // api key in headers '{{keyParamName}}'
-    req.headers.upsert("{{{name}}}", api_key['{{{keyParamName}}}'])
+    -- api key in headers '{{keyParamName}}'
+    req.headers:upsert("{{{name}}}", api_key['{{{keyParamName}}}'])
     {{/isKeyInHeader}}
     {{#isKeyInQuery}}
-    /// TODO: api key in query '{{keyParamName}}'
+    -- TODO: api key in query '{{keyParamName}}'
     {{/isKeyInQuery}}
     {{/isApiKey}}
     {{#isBasic}}
-    // HTTP basic auth
-    req.headers.upsert("Authorization", base64_encode(self.http_username .. " " .. self.http_password))
+    -- HTTP basic auth
+    req.readers:upsert("Authorization", "Basic " .. basexx.to_base64(self.http_username .. " " .. self.http_password))
     {{/isBasic}}
     {{#isOAuth}}
-    // oAuth
-    req.headers.upsert("Authorization", "Bearer " .. self.access_token)
+    -- oAuth
+    req.headers:upsert("Authorization", "Bearer " .. self.access_token)
```

Comment:
Based on the examples I found (e.g. https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication). "Authorization" (header key) is usually in upper case. (HTTP header key is case-insensitive if I'm not mistaken)